demisto_sdk/commands/create_id_set/tests/id_set_creator_test.py

```diff
@@ -47,3 +49,34 @@ def test_create_id_set_no_output(self):
         assert 'Layouts' in id_set.keys()
         assert 'Reports' in id_set.keys()
         assert 'Widgets' in id_set.keys()
+
+
+INPUT_TEST_HAS_DUPLICATE = [
+    ('Access', False),
+    ('urlRep', True)
+]
+
+ID_SET = [
+    {'Access': {'typeID': 'Access', 'kind': 'edit', 'path': 'Layouts/layout-edit-Access.json'}},
+    {'Access': {'typeID': 'Access', 'fromversion': '4.1.0', 'kind': 'details', 'path': 'layout-Access.json'}},
+    {'urlRep': {'typeID': 'urlRep', 'kind': 'Details', 'path': 'Layouts/layout-Details-url.json'}},
+    {'urlRep': {'typeID': 'urlRep', 'fromversion': '5.0.0', 'kind': 'Details', 'path': 'layout-Details-url_5.4.9.json'}}
+]
+
+
+@pytest.mark.parametrize('list_input, list_output', INPUT_TEST_HAS_DUPLICATE)
+def test_has_duplicate(list_input, list_output):
```

Comment:
Add doc string for the test (given, when, then). Look at the upload command unit tests file.